### ISSUE TITLE
Add icons for certain situations

### ISSUE THREAD

**Issue by Neriderc** [Neriderc/GVExport#418]
As discussed in https://github.com/Neriderc/GVExport/issues/397

We could add icons for various situations:
- a mourning ribbon to indicate dead individuals
- a crown as a symbol for persons with a noble title
- colourful balloons when a birthday/wedding anniversary is coming up

These should appear at specific places:
- left-top-corner: mourning ribbon
- balloons/birthday cake behind the tile right-top-corner
- maybe other coloured markers inside the tile on the right side

Examples from MyHeritage:
![Screenshot (296) - Kopie](https://github.com/Neriderc/GVExport/assets/6324175/5c096aa9-c0b3-4bcf-9453-5b70870ead60)
![Screenshot (297)](https://github.com/Neriderc/GVExport/assets/6324175/8b2bcbdc-f739-4a2a-80b4-4a1ce9b3093d)
![Screenshot (296)](https://github.com/Neriderc/GVExport/assets/6324175/ac708613-d7db-42ad-8cc6-03125dc211fb)

**Comment by Neriderc** [comment]
@hartenthaler I suspect your ideas for placement are not going to play nice with Graphviz, considering all the trouble we have had positioning things much more basic than this. 

I think this is on the hard end of the scale compared to the other things we have discussed, but when I get to it, I will have a play and try to find something that has the same idea but works nicer within the table format that we use with Graphviz.

**Comment by hartenthaler** [comment]
Maybe you have seen that the Vesta custom module "Classic Look & Feel" now supports "name badges", ie small icons attached to the name, based on tags (or more general: information found by regular expressions). So showing static information (like "a crown as a symbol for persons with a noble title") is solved now. The other examples are more dynamic and not static. Do you think that the new Graphviz engine will allow a new sight on this issue? But again: the priority of this issue is low.

**Comment by Neriderc** [comment]
Ultimately the DOT rendering engine works the same, and we are still working within the same "HTML-like" table tags to build our person tile. In the examples above, I think the birthday cake and black ribbon are probably not possible without some major work. 

However, I do think _something_ should be possible. You mentioned adding the icons to the name, this could be easier, it may even be easy if we stick to unicode characters/emojis so they are treated like text.

I'm also thinking the dots down the side may be possible, but that would require some experimenting.

For the more basic options, the hardest bit is probably working out the UI. We now have a shared note system that could be used, and it seems like an extension of this would be the best option, but for attributes like death this should not be based on a shared note. Some thought will be needed on the best way to display this.

**Comment by Neriderc** [comment]
@hartenthaler just having another look at this. I'm wondering if there are some fundamental questions here about what it is we are trying to do with GVExport. Is it supposed to be for generating diagrams for exporting, or is it supposed to be for a main workspace? With the current setup, things we add in the browser diagram will end up in the exported diagram.

So, as an example, we could add a birthday cake for people if it's their birthday today. However, this changes the fundamental intent of the tool, as now you will get a different diagram depending on what day you generate it.

Something like a crown for noble titles is more consistent, but it seems you have already solved this one using another module.

Do you (or others) have any thoughts about this?

**Comment by schuco** [comment]
I agree that GVE should not aim on serving birthday cakes.
For me there are mainly two purpose where I use it:

1. generating diagrams for exporting
2. user interface to navigate withon webtrees

Many of the features you and others have added like full screen workablility, colours based on shared notes, saved setting  etc.  proved to be very helpful

**Comment by Neriderc** [comment]
Given the discussion above, is there anything here to be done?

Some items are just not possible with the Graphviz tool, e.g. mourning ribbon over the corner

Others can be added as icons in the name of the individual within webtrees, which flows through to the diagram.

Can this issue be closed or is there still something here to do?

**Comment by hartenthaler** [comment]
Yes, close it. Using the color options based on shared notes (tags) is such a rich function of GVExport. That allows to visualize many things.
